app/main.py

```diff
@@ -1,8 +1,51 @@
+def limitation(value: float,
+               min_value: float,
+               max_value: float) -> float:
+    return max(min(value, max_value), min_value)
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = int(limitation(comfort_class, 1, 7))
+        self.clean_mark = int(limitation(clean_mark, 1, 10))
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = limitation(
+            distance_from_city_center, 1.0, 10.0)
+        self.clean_power = int(limitation(clean_power, 1, 10))
```

Comment:
Why do you convert to int? I mean what is bad about comfort class is float for example?(as I remember it's always int in this task). Like you can still work with float